Cargo.toml

```diff
@@ -62,6 +62,7 @@ xmltree = "0.10.0"
 hyper = { version = "1.1.0", optional = true }
 warp = { version = "0.3.0", optional = true, default-features = false }
 actix-web = { version = "4.0.0-beta.15", optional = true }
+reflink = "0.1.3"
```

Comment:
Please make it optional and only enable it when `localfs` feature is activated.

Reply:
Unfortunately that crate doesn't support musl: https://github.com/nicokoch/reflink/issues/11, we'll need to find an alternative maintained crate or vendor the code.

Reply:
https://github.com/cargo-bins/reflink-copy is a maintained alternative, apparently solving the problem in a different way: cargo-bins/reflink-copy#30 (in version 0.1.8 already). 281f21eb6455196d8ed3ee9f6ef4f8912000fec5 tries to address this. It seemed to me that the releases since 0.1.14 are mostly updating dependencies; the last real feature was cargo-bins/reflink-copy#57; that is why I chose it instead of the currently latest release 0.1.19.